### PR TITLE
Fixing the database concept

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

Applying fixes already used in 7.9.x branch.

### CODE REVIEW

**Comment by MichalMaler** [comment]
Give it a sec Yana! Robert found some more stuff to fix

File: src/main/pages/che-7/administration-guide/con_database-setup.adoc [hunk 1]
```diff
@@ -14,18 +14,18 @@ summary:
 [id="database-setup_{context}"]
 = Database setup
 
-PostgreSQL database is used for persisting data about the state of {prod-short}. It contains information about the user accounts, workspaces, preferences, ssh keys, and so on.
+PostgreSQL database is used by Che server for persisting data about the state of {prod-short}. It contains information about the user accounts, workspaces, preferences, and so on.
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/con_database-setup.adoc]:
```suggestion
The PostgreSQL database is used by the Che server for persisting data about the state of {prod-short}. It contains information about user accounts, workspaces, preferences, and so on.
```

File: src/main/pages/che-7/administration-guide/con_database-setup.adoc [hunk 2]
```diff
@@ -14,18 +14,18 @@ summary:
...
 
-By default, {prod-short} operator creates and manages a database deployment. 
+By default, {prod-short} operator creates and manages a database deployment.
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/con_database-setup.adoc]:
```suggestion
By default, the {prod-short} Operator creates and manages the database deployment.
```

File: src/main/pages/che-7/administration-guide/con_database-setup.adoc [hunk 3]
```diff
@@ -14,18 +14,18 @@ summary:
...
 
 However, {prod-short} operator does not support the full lifecycle capabilities like backups and recovery. For business-critical setup, it is recommended to configure an external database with the following disaster recovery options:
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/con_database-setup.adoc]:
```suggestion
However, the {prod-short} Operator does not support full life-cycle capabilities, such as backups and recovery. For a business-critical setup, configure an external database with the following recommended disaster recovery options:
```
"life-cycle" -- IBM SG, pg. 341

File: src/main/pages/che-7/administration-guide/con_database-setup.adoc [hunk 4]
```diff
@@ -14,18 +14,18 @@ summary:
...
 
 * High Availability (HA)
 * Point In Time Recovery (PITR)
 
-You can configure External PostgreSQL instance on-premises or using cloud services such as Amazon Relational Database Service (Amazon RDS). With Amazon RDS, you can deploy production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on-demand snapshots.
+Configure an External PostgreSQL instance on-premises or use a cloud services such as Amazon Relational Database Service (Amazon RDS). With Amazon RDS, it is possible to deploy production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on-demand snapshots.
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/con_database-setup.adoc]:
```suggestion
Configure an external PostgreSQL instance on-premises or use a cloud service, such as Amazon Relational Database Service (Amazon RDS). With Amazon RDS, it is possible to deploy production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on demand snapshots.
```
"on demand -- IBM SG, pg. 350

**Review by rkratky — CHANGES_REQUESTED** [review]
@MichalMaler, suggestions added.

**Review by rkratky — APPROVED** [review]
Thanks, @MichalMaler

**Pull request merged**